data/trie/statistics/trieSyncStatistics.go

```diff
@@ -59,6 +68,14 @@ func (tss *trieSyncStatistics) NumReceived() int {
 	return tss.numReceived
 }
 
+// NumLarge returns the large received nodes
```

Comment:
NumLarge returns the received large nodes

Reply:
done